### DISCUSSION TITLE
How can I port/develop an app for Google Android, Web and Microsoft Appsource and Marketplace App Stores

### DISCUSSION THREAD

**Discussion by MsD201x** [freeplane/freeplane#591] [Ideas]
**My Problem**:  Frustrated in my learning Freeplane. Why? On desktop, I am ok. But, if on the web at Google, Microsoft apps marketplace, Freeplane isn't there. Freeplane is not an optional app for me and my Freeplane learning development.
**My Solution**:  I want to develop an app for Google android, web and Microsoft appsource and marketplace app stores.

**Comment by ldmpub** [comment]
Hi,

Please do not open several Discussions for exactly the same subject (see #592).
I suggest you to close this one 💪.

PS : I will answer you there #592 😉
